pkgs/by-name/du/duckstation/package.nix

```diff
@@ -57,27 +65,34 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = [
     SDL2
     curl
+    extra-cmake-modules
     libXrandr
     libbacktrace
+    libwebp
     qtbase
     qtsvg
     qtwayland
+    shaderc-patched
     wayland
   ]
   ++ cubeb.passthru.backendLibs;
 
-  strictDeps = true;
-
   cmakeFlags = [
     (lib.cmakeBool "BUILD_TESTS" true)
   ];
 
-  doCheck = true;
-  checkPhase = ''
-    runHook preCheck
-    bin/common-tests
-    runHook postCheck
-  '';
+  qtWrapperArgs =
+    let
+      libPath = lib.makeLibraryPath ([
+        vulkan-loader
+      ] ++ cubeb.passthru.backendLibs);
+    in [
+      "--prefix LD_LIBRARY_PATH : ${libPath}"
+    ];
```

Comment:
Same as this change - you just moved it, and you enlarge the diff without no real change to the derivation.. I usually make an effort to make the amount of diffs as small as possible.

Reply:
I split this in a separate commit.

Reply:
> I split this in a separate commit.

That's good idea, but still, now in the 3rd commit, the `qtWrapperArgs` and the `installCheckPhase` switched places after the reformat commit for no real reason.

Reply:
Hammering:

https://github.com/jtojnar/nixpkgs-hammering